src/fsharp/range.fs

```diff
@@ -200,6 +200,10 @@ let fileOfFileIndex idx = fileIndexTable.IndexToFile idx
 
 let mkPos l c = pos (l, c)
 
+let unknownFileName = "unknown"
```

Comment:
This is fine. Just for future reference, I would make these `[<Literal>]` but we still have a bug for `fsi`/signature files that prevents using `[<Literal>]` in a reasonable way. Therefore, we can't make these literal *yet*.

Reply:
Yes, I tried to make it `[<Literal>]` at first and had to report the bug instead in https://github.com/dotnet/fsharp/issues/7897.

Reply:
Great, thanks!